src/types.ts

```diff
@@ -68,3 +69,12 @@ export type SubscribePsuedoIterable = {
 export type SubscribeArgs = any[];
 
 export type Class = { new (...args: any[]): any };
+
+export interface APIGatewayWebSocketRequestContext extends APIGatewayEventRequestContext {
+  connectionId: string
+  domainName: string
+}
+
+export interface APIGatewayWebSocketEvent extends APIGatewayProxyEvent {
+  requestContext: APIGatewayWebSocketRequestContext
+}
```

Comment:
I should have done this earlier - noice :rocket: 

Any particular reason for using `APIGatewayProxyEvent` vs `APIGatewayEvent`?

Might be worth squeezing this into one type and using functional operators so types don't get out of sync.

```tsx
export type APIGatewayWebSocketEvent = APIGatewayEvent & {
  requestContext: APIGatewayEventRequestContext & Required<Pick<APIGatewayEventRequestContext, 'connectionId'>>
};
```